modules/api/php/views/project.class.inc

```diff
@@ -51,12 +52,21 @@ class Project
     {
         $meta = ['Project' => $this->_project->getName()];
 
-        return [
+        $obj = [
             'Meta'        => $meta,
             'Candidates'  => $this->_project->getCandidateIds(),
             'Instruments' => array_keys(\Utility::getAllInstruments()),
             'Visits'      => $this->_getVisits(),
         ];
+        if ($this->apiversion == 'v0.0.4-dev') {
```

Comment:
Should we use this so w don't have to go over every version checks each time we have a new version?

```
if ($this->apiversion >= 'v0.0.4') {
```

Reply:
I don't think that will work since 'v0.0.4' is a string, but I can make it a switch to make it easier to add new versions

Reply:
It would work but there is issues when/if we get to 2 digits parts
~$ php -r 'var_dump("v1.0.14" <= "v1.0.5");'
bool(true)